sales/sales.js

```diff
@@ -1,99 +1,83 @@
-var express = require('express');
-var http = require('http');
-var os = require('os');
+'use strict';
 
-var salesData = require('./data');
-var consul = require('./consul');
+const Consulite = require('consulite');
+const Express = require('express');
+const Customers = require('./lib/customers');
+const Data = require('./lib/data');
 
-var app = express();
-var customersHosts = [];
+const app = Express();
```

Comment:
Hrm, one of the big advantages of this blueprint relative to the others is that the whole thing is pretty small and easy to understand. Adding more services and dependencies and complexity doesn't jive with that.